### PR TITLE
Rework the README.md

### PR DESCRIPTION
**Pull request by Samahu**
Linked PR: None
Linked Issues:
* https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-549

### CODE REVIEW

**Review by jfugate1 — APPROVED** [review]

**Review by jfugate1 — APPROVED** [review]

File: ow_lander/README.md [hunk 1]
```diff
@@ -1,84 +1,149 @@
-The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+> The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
```
Comment (kmdalal) [inline ow_lander/README.md:1]: Is the '>' a typo or this Markdown syntax?

File: ow_lander/README.md [hunk 2]
```diff
@@ -1,84 +1,149 @@
...
+URDF and related code. Please prefer this header over writing similar code
+yourself to keep all our code consistent. If the lander is modified to change
+the number of joints or any joint names, this header must be updated as well.
+
+### Package History
+
+**Current Version:**
+A major overhaul of this package was done after switching from JointPositionController
+to JointPositionController for arm services. Nonetheless, this revised version
+is more comptabile with the file generated by the setup_assistant with few manual
```
Comment (kmdalal) [inline ow_lander/README.md:143]: to JointTrajectoryController  ?

File: ow_lander/README.md [hunk 3]
```diff
@@ -1,84 +1,149 @@
...
+
+## Misc Information
+
+### Common Code
+`include/ow_lander/lander_joints.h` contains joint names as they appear in the
+URDF and related code. Please prefer this header over writing similar code
+yourself to keep all our code consistent. If the lander is modified to change
+the number of joints or any joint names, this header must be updated as well.
+
+### Package History
```
Comment (kmdalal) [inline ow_lander/README.md:138]: Is the Package History section really helpful, especially in the public code?  Are any users likely to care or be impacted by the changes?

File: ow_lander/README.md (hunk 1 above)
Comment (kmdalal) [reply ow_lander/README.md:1]: I now see that this section appears like an email quote, which looks odd to me...

File: ow_lander/README.md [hunk 4]
```diff
@@ -1,84 +1,149 @@
...
-Call the DeliverSample service.
+### Calling Arm Operations using ROS Command Line Interface
+Alternatively to using the **RQT Service Caller** gui interface, you may interact
+with the services using the `rosservice` command line tool. While the simulation
+is running, start a new terminal and source OceanWATERS workspace. Then, invoke
+the services using their respective commands as listed below:
 
-### Stow trajectory planning
-Call the Stow service.
+> Note: The provide examples show running these service using default values.
```
Comment (kmdalal) [inline ow_lander/README.md:57]: Minor comment, but I find the '>' formatting distracting.  I think the document would look better without it.

File: ow_lander/README.md [hunk 5]
```diff
@@ -1,84 +1,149 @@
...
+rosservice call /arm/stow "{}"
+```
+* Unstow
+```bash
+rosservice call /arm/unstow "{}"
+```
+* Deliver Sample
+```bash
+rosservice call /arm/deliver_sample \
+  "{use_defaults: true,
```
Comment (kmdalal) [inline ow_lander/README.md:70]: I don't understand why, but every 'use_defaults: true' line appears red in my Markdown viewer.

**Review by kmdalal — APPROVED** [review]
The content is good, but over-formatted for my personal taste, and the Package History seems unnecessary, though fine to leave in if you see it helpful.

File: ow_lander/README.md (hunk 1 above)
Comment (Samahu) [reply ow_lander/README.md:1]: It is a Markdown syntax. On my markdown viewer and github the paragraph would appear grayed out.

File: ow_lander/README.md (hunk 2 above)
Comment (Samahu) [reply ow_lander/README.md:143]: Correct, I will fix it.

File: ow_lander/README.md (hunk 3 above)
Comment (Samahu) [reply ow_lander/README.md:138]: I am with you one on this. I wasn't sure if other think it is helpful or not, personally I didn't read much through it, it seemed like our meeting notes. I am in favor of removing it.

File: ow_lander/README.md (hunk 4 above)
Comment (Samahu) [reply ow_lander/README.md:57]: I used to add this '>' to notes, just to highlight them from normal texts
> Note: this is a note

File: ow_lander/README.md (hunk 5 above)
Comment (Samahu) [reply ow_lander/README.md:70]: I don't have this issue, change your markdown viewer :wink:

Comment (mogumbo) [reply ow_lander/README.md:69]: I don't have this issue using the Markdown Viewer extension in Chrome.

File: ow_lander/README.md (hunk 3 above)
Comment (mogumbo) [reply ow_lander/README.md:137]: It's a good practice for automatically generated files to begin with a comment saying they are automatically generated. MoveIt Setup Assistant generates files that do not contain such a comment. This part of the README contains information about which files were generated and which of those have been modified by hand, which could be useful information if we need to re-run Setup Assistant. It should be revised due to all the changes in ow_lander, but probably not deleted.

Comment (Samahu) [reply ow_lander/README.md:137]: Terry, that's not the case here. This is a wizard generated files and they serve as a template but we are free to modify. What you are referring to are auto-generated that we shouldn't modify.

Comment (Samahu) [reply ow_lander/README.md:137]: In any case, I left a link to this OCEANWATER-518 PR that we can refer to later in case we needed to.

Comment (mogumbo) [reply ow_lander/README.md:137]: If we re-generate files, any mods we made would be stomped. It's good to have an understanding of what mods we made so they can be added back in. Of course, it's easiest if we can avoid modifying generated files.

**Comment by Samahu** [comment]
@mikedalal Since your markdown viewer isn't working properly, you can see a rendered version of the file [here](https://github.com/nasa/ow_simulator/blob/e7ad4a7741768075a9217cc761f0041f9dc77ebb/ow_lander/README.md). I mean how it looks on GitHub that's what should we care about.

File: ow_lander/README.md (hunk 3 above)
Comment (Samahu) [reply ow_lander/README.md:137]: Setup assistant does that for you. I mean it shows you which files are new and which files are going to be modified before you save the new config, plus all of our files are version tracked using git.

**Review by mogumbo — APPROVED** [review]

**Comment by kmdalal** [comment]
> @mikedalal Since your markdown viewer isn't working properly, you can see a rendered version of the file [here](https://github.com/nasa/ow_simulator/blob/e7ad4a7741768075a9217cc761f0041f9dc77ebb/ow_lander/README.md). I mean how it looks on GitHub that's what should we care about.

Thanks, I'll just look at these in GitHub from now on, since that's the definitive viewer.  I had opened the file in GitHub but it showed unformatted.  I presume I need to switch to the branch itself.

**Pull request merged**